cmd/validator/core/command.go

```diff
@@ -43,14 +44,15 @@ func init() {
 	ValidatorCmd.Flags().IntVarP(&config.Retries, "retries", "", 5, "Number of retry attempts when a difference is detected.")
 	ValidatorCmd.Flags().IntVarP(&config.RetryDelayMS, "retry-delay", "", 1000, "Time in milliseconds to sleep between retries.")
 	ValidatorCmd.Flags().StringVar(&addr, "addr", "", "If provided validate a single address instead of reading Stdin.")
+	ValidatorCmd.Flags().StringVar(&box, "box", "", "If provided validate a single (appid,b64boxname) instead of reading Stdin.")
```

Comment:
```suggestion
	ValidatorCmd.Flags().StringVar(&box, "box", "", "If provided validate a single box (in the format "appid,b64boxname") instead of reading Stdin.")
```